style.css

```diff
@@ -333,3 +335,98 @@ body {
 hr {
   margin: 0;
 }
+
+.contact-me {
+  border-radius: 25px 0 0 0;
+  border: 1px;
+  background-color: #6070ff;
+  top: 70px;
+  color: #fff;
+  text-align: center;
+  padding: 70px 16px 20px 16px;
+}
+
+.dot1 {
+  position: relative;
+  top: -30px;
+}
+
+.contactme-header {
+  color: #fff;
+  padding-bottom: 20px;
+  font-size: 40px;
+}
+
+.contactme-p {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: #fff;
+  padding-bottom: 20px;
+}
+
+#name {
+  height: 48px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  font-size: 17px;
+  line-height: 20px;
+  font-weight: 400;
+}
+
+#mssg {
+  height: 170px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+#mssg::placeholder {
+  text-align: left;
+  position: absolute;
+  top: 15px;
+  bottom: 15px;
+  left: 16px;
+  right: 38px;
+}
+
+.getintouch {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  background-color: #fff;
+  border-radius: 8px;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  margin-top: 5px;
+  margin-bottom: 30px;
+  border: 1px solid #7f8cff;
+}
+
```

Comment:
- [ ] You have done incredibly well to ensure the button changes the background color when active 👏 . As a way of making sure you try as much as possible to stick to the Figma template design, kindly ensure that other button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template ☺️ .

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220330939-9d5e79f5-fc46-41fb-beec-a63f5703c984.png)